docs/data/whats-new.md

```diff
@@ -1,4 +1,4 @@
-# ✨ What's new in v6? ✨
+# ✨ What's new in MUI X v6? ✨
```

Comment:
Should we also do the same for the navbar link?

Reply:
That's a good point; I tried it out. 
But I think the menu item simplicity (without MUI X) feels more appealing aesthetic.
It may also be close enough to the product name in the top-left corner, so perhaps we don't need it.

I made a side-by-side comparison experiment. Let me know what you think.

<img width="604" alt="image" src="https://user-images.githubusercontent.com/550141/217058888-dbd2f136-adbd-4aa0-8351-0f0da116f808.png">

Reply:
The one on the left is more explicit, but the one on the right looks better :+1:
Let's keep it as it is now